templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -4,35 +4,116 @@ contents:
   inline: |
     #!/bin/bash
     set -eux
-    
+
     # This file is not needed anymore in 4.7+, but when rolling back to 4.6
     # the ovs pod needs it to know ovs is running on the host.
     touch /var/run/ovs-config-executed
 
+    NM_CONN_OVERLAY="/etc/NetworkManager/system-connections-merged"
```

Comment:
This is wrong on 4.7

Red Hat Enterprise Linux CoreOS 47.84.202109210831-0

```
drwxr-xr-x. 2 root root system_u:object_r:NetworkManager_etc_rw_t:s0 146 Sep 22 02:55 /etc/NetworkManager/system-connections
drwxr-xr-x. 1 root root system_u:object_r:var_run_t:s0               160 Sep 22 02:46 /etc/NetworkManager/systemConnectionsMerged
```

Reply:
ah keyfile can't load it because the path is wrong.

```ini
# cat /etc/NetworkManager/conf.d/99-keyfiles.conf
[main]
plugins=keyfile,ifcfg-rh
[keyfile]
path=/etc/NetworkManager/systemConnectionsMerged
```

Reply:
Fixed